AppOne/AppOne/SceneDelegate.swift

```diff
@@ -17,6 +17,9 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         // If using a storyboard, the `window` property will automatically be initialized and attached to the scene.
         // This delegate does not imply the connecting scene or session are new (see `application:configurationForConnectingSceneSession` instead).
         guard let _ = (scene as? UIWindowScene) else { return }
+        window = UIWindow()
```

Comment:
actually you should give the `windowScene` here to the `window`, over it"s initializer

Reply:
@JoeHoHH Updated [here](https://github.com/JoeHoHH/ajayTheLead/pull/1/commits/82b81a4276343fca98befc7014edae74c8e77fa8)